vllm/entrypoints/openai/serving_chat.py

```diff
@@ -113,39 +112,36 @@ def _parse_chat_message_content_parts(
             part_type = part["type"]
             if part_type == "text":
                 text = cast(ChatCompletionContentPartTextParam, part)["text"]
-
                 texts.append(text)
             elif part_type == "image_url":
                 if vlm_config is None:
                     raise ValueError(
                         "'image_url' input is not supported as the loaded "
                         "model is not multimodal.")
+                assert self.tokenizer is not None
```

Comment:
Is it intended that you removed the `elif` block?

Reply:
yeah I am checking for multiple images case down below at line 144.